ArduCopter/mode_autotune.cpp

```diff
@@ -82,8 +82,8 @@ void AutoTune::get_pilot_desired_rp_yrate_rad(float &des_roll_rad, float &des_pi
 void AutoTune::init_z_limits()
 {
     // set vertical speed and acceleration limits
-    copter.pos_control->set_max_speed_accel_U_cm(-copter.get_pilot_speed_dn(), copter.g.pilot_speed_up, copter.g.pilot_accel_z);
-    copter.pos_control->set_correction_speed_accel_U_cm(-copter.get_pilot_speed_dn(), copter.g.pilot_speed_up, copter.g.pilot_accel_z);
+    copter.pos_control->set_max_speed_accel_U_m(-copter.flightmode->get_pilot_speed_dn_ms(), copter.flightmode->get_pilot_speed_up_ms(), copter.flightmode->get_pilot_accel_mss());
```

Comment:
it's a bit sad that we need to do this but the only way to avoid it would be to create more function within the Copter class so this is actually OK I think.

Reply:
Sorry, I think we need to create the extra functions in the Copter class :-(

Reply:
Actually we could replace, "copter.flightmode->" with "copter.autotune." which would remove the danger of the nullptr.

@peterbarker what do you think?

Reply:
> Actually we could replace, "copter.flightmode->" with "copter.mode_autotune." which would remove the danger of the nullptr.
> 
> @peterbarker what do you think?

`copter.flightmode` will never be nullptr - if it is then we will have fallen out of the sky before getting to this point.

`copter.mode_autotune.` vs `copter.flightmode->` ... I think flightmode-> is actually more correct in that someone may subclass ModeAutoTune - in which case calling the method on `copter.mode_autotune` wouldn't be the right thing to do.